### ISSUE TITLE
Inconsistent selectors for type and keys

### ISSUE THREAD

**Issue by maba4891** [laravel/dusk#523]
It seems like I can use an input's name as a selector with ->type() but not with ->keys().

I have two input fields:
```
<input name="name" type="text">
<input name="date" type="date">
```

E.g. the following is working:
` ->type('name', 'John')`

But the following is not working:
`->keys('date', '10101990')`

Instead it returns the following error:
```
1) Tests\Browser\PRTest::testSimpleCreate
Facebook\WebDriver\Exception\InvalidSelectorException: invalid selector: An invalid or illegal selector was specified
  (Session info: chrome=67.0.3396.62)
  (Driver info: chromedriver=2.35.528139 (47ead77cb35ad2a9a83248b292151462a66cd881),platform=Linux 4.15.0-22-generic x86_64)
```

If I use the full selector it is working:
`->keys('input[name=date]', '10101990')`

The reason why I have to use keys() instead of type() is because the second input is a date field (see #104 ).

To summarize: I find it very confusing that type() is accepting just the name while keys() seems to require the full selector. Is this intended behavior or am I missing something?

**Comment by driesvints** [comment]
I think it's indeed the same issue as #104, however since this is probably a chromedriver bug this isn't really a Dusk issue so therefor I'm going to close this. Let's hope this will be solved in a future chromedriver release.
